config.yml

```diff
@@ -496,12 +496,20 @@ nodes:
       - name: flags
         type: flags
         kind: ArrayNodeFlags
+        comment: |
+          Represents whether or not there is a splat operator in the array.
       - name: elements
         type: node[]
+        comment: |
+          Represent list of elements in the array.
```

Comment:
More information needed here:

* What kind of nodes can be present in this list?
* Is it every possible node or are there some restrictions?
* Will there always be at least one element or can this be empty?

Reply:
I don't really have much idea about the restrictions on the types of nodes that can be present in the list. Can you provide any source for reference?

Reply:
I don't have one at the moment, unfortunately. That's why we want to do this documentation effort!

Reply:
I'm currently facing a challenge; I've manually tested a few scenarios where I had some uncertainties. Do you have any suggestions or ideas for testing which nodes can be contained within the elements? For instance, based on my understanding, a `WhenNode` cannot be directly placed in the array; it always needs to be nested under a `CaseNode`